apps/etterlatte-saksbehandling-ui/client/src/components/behandling/avkorting/Avkorting.tsx

```diff
@@ -105,24 +107,28 @@ export const Avkorting = ({
                 redigerbar={redigerbar}
                 resetInntektsavkortingValidering={resetInntektsavkortingValidering}
               />{' '}
-              {toggleAvkortinGrunnlagNesteAar &&
-                behandling.behandlingType === IBehandlingsType.FØRSTEGANGSBEHANDLING && (
-                  <AvkortingInntekt
-                    behandling={behandling}
-                    avkortingGrunnlagFrontend={avkorting?.avkortingGrunnlag[1]}
-                    erInnevaerendeAar={false}
-                    redigerbar={redigerbar}
-                    resetInntektsavkortingValidering={resetInntektsavkortingValidering}
-                  />
-                )}
+              {behandling.behandlingType === IBehandlingsType.FØRSTEGANGSBEHANDLING && (
+                <AvkortingInntekt
```

Comment:
Skal denne vises for alle førstegangsbehandlinger?

Reply:
Den er ikke så relevant hele året men utenom det så skal den jo det